src/state/notificationStore.ts

```diff
@@ -1,13 +1,55 @@
 import { create } from 'zustand'
+import { devtools, persist } from 'zustand/middleware'
+import { Notification } from '@prisma/client'
 
-type Notification = {
-  id: number
-  message: string
-  // type: 'success' | 'error' | 'info';
-}
+type INotification = Omit<Omit<Notification, 'createdAt'>, 'updatedAt'>
 
-type NotificationStore = {
-  notifications: Notification[]
-  addNotification: (notification: Notification) => void
-  removeNotification: (id: number) => void
+interface Notifications {
+  [modelId: string]: INotification
+}
+type Store = {
+  notification: INotification[]
+  Notifications?: Notifications
+  addNotification: (notification: INotification) => void
+  markAsRead: (id: string) => void
 }
+
+const useNotificationStore = create<Store>()(
+  persist(
+    devtools((set, get) => ({
+      notification: [],
+
```

Comment:
can u init the notificaiton store with the dummy data. And use the notification store in Notification.tsx instead of '@/data/notification''